crawl-ref/source/dat/des/variable/float.des

```diff
@@ -9930,3 +9957,138 @@ x..x..x11xx...........xx11x..x..x
 x1.x.1x11*xxx...{...xxx*11x1.x.1x
 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 ENDMAP
+
+NAME:   amcnicky_float_guardroom
+DEPTH:  D:3-, Depths
+ORIENT: float
+TAGS:   no_item_gen no_monster_gen
+NSUBST:  0 = 4:. / 8:0 / 6:09 / 2:98
+#Total weight 100
+NSUBST:  % = *=.:20 $:40 %:26 *:10 |:4
+NSUBST:  p = 1:+ / *=.+
+MAP
+    xxxxxxxxxx
+    x%%999%%%x
+  ..x........x
+  ..x00....00x
+xxxpx00....00x
+x00.xxxx+xxxxxxx
+x00............x
+xxxxxxpxxxxxxxpx
+           x00.x
+           x00.x
+           x00..
+           xx...
+            xx..
+ENDMAP
+
+NAME:   amcnicky_float_beach_large
+WEIGHT: 5 (D:3-), 2 (Depths)
+DEPTH:  D:3-, Depths
+ORIENT: float
+TAGS:   no_item_gen no_monster_gen
+FTILE:  . = floor_sand
+FTILE:  ; = floor_pebble
+FTILE:  0 = floor_sand
+FTILE:  9 = floor_sand
+FTILE:  % = floor_sand
+NSUBST:  0 = 4:. / 4:0 / 1:09 / 1:98
+#Total weight 100
+NSUBST:  % = *=.:20 $:40 %:26 *:10 |:4
+NSUBST:  p = 1:+ / *=.+
+SUBST:   q : .+x
+MAP
+  xxxx
+  +;;x
+  xx;x
+xxxx;xxxx
+x..0.0..x
+x.0.0.0.x
+q..www..x
+q.0www..x
+q..www..x
+x0.www..x
+x..www9.x
+x.0www..x
+x..www..x
+q0.www..x
+q..www..x
+q.0...%%x
+x.....%%x
+xxxxxxxxx
+ENDMAP
+
+NAME:   amcnicky_float_corridor_feature
+DEPTH:  D:5-, Depths, Crypt
+ORIENT: float
+TAGS:   extra no_item_gen transparent
+# Total weight: 100
+SUBST:  p: .:6 G:40 t:20 <:10 >:10 n:4 l:4 $:4 Y:2
+MAP
+xx@xx
+xp.px
+xx.xx
+xx.xx
+xp.px
+xx.xx
+xx.xx
+xp.px
+xx@xx
+ENDMAP
+
+# Vault walls and blood hint that something tougher awaits
+NAME:   amcnicky_float_rare_duel
```

Comment:
This would be the start of a good minivault storyline concept if its tactical options were more interesting than a corridor with door.